bin/fetch_github_actions_quota.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import logging
+
+from services.github_service import GithubService
+from services.kpi_service import KpiService
+from config.constants import ENTERPRISE, MINISTRY_OF_JUSTICE
+
+
+def get_environment_variables() -> str:
+    github_token = os.getenv("GH_TOKEN")
+    if not github_token:
+        raise ValueError(
+            "The env variable GH_TOKEN is empty or missing")
+
+    return github_token
+
+
+def fetch_gha_quota():
+    github_token = get_environment_variables()
+    if github_token is None:
+        print("No GH_TOKEN environment variable set")
+        sys.exit(1)
+
+    github_service = GithubService(github_token, MINISTRY_OF_JUSTICE, ENTERPRISE)
+
+    # Get all organisations
+    organisations = github_service.get_all_organisations_in_enterprise()
+    # Get total quota usage
+    total_minutes_used = github_service.calculate_total_minutes_used(organisations)
+
+    # Save metric in KPI dashboard db
+    try:
```

Comment:
Can probably remove the try/catch in this instance since the scripts job is to store the data in the KPI Dashboard 👀 

The reason we have it in the other scripts is because they're primary purpose is typically something else and storing the data in the KPI Dashboard is a side-affect 😁

Reply:
Cool, removed.